code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -158,7 +158,7 @@
 
 /obj/item/dnainjector/hulkmut/Initialize()
 	block = GLOB.hulkblock
-	..()
+	. = ..()
```

Comment:
Might be out of the scope of this PR, but could you define a `getBlock()` proc for injectors and have the parent call use that akin to `populateContents()` on a closet?

The fact the super call is second here instead of first is killing me.

Reply:
Done